src/interpret.jl

```diff
@@ -12,6 +12,12 @@ function sbmlPiecewise(args...)
 end
 
 sbmlNeq(a, b) = !isequal(a, b)
+sbmlRelational(fun) = begin
+    function f(x...)
+        all(map(i -> fun(x[i], x[i+1]), 1:length(x)-1))
+    end
+    f
+end
```

Comment:
I think this can be simplified (julia `map` works nicely as `zipWith`):
```suggestion
sbmlRelational(op) =
    (x...) -> all(map(op, x[begin:end-1], x[begin+1:end]))
```